ocs_ci/ocs/exceptions.py

```diff
@@ -678,3 +678,7 @@ class HostValidationFailed(Exception):
 
 class SameNameClusterAlreadyExistsException(Exception):
     pass
+
+
+class NoRunningCephToolBoxException(Exception):
```

Comment:
we already have CephToolBoxNotFoundException, you can use the existing one

Reply:
that error is for cephtoolbox pod not existing. This is for tool pod is not in status running